### DISCUSSION TITLE
Why vercel serverless functions don't work correctly with Vuejs?

### DISCUSSION THREAD

**Discussion by sidiDev** [vercel/community#1436] [Help]
Hi everyone!

The few days ago I was working on a Vue app, and I tried to integrate the serverless functions, but didn't work correctly with me.

When I create an api route that return a json message like: 
api/test: this file includes res.json({success: true})  and deploy the app and go to domain.vercel.app/test I get the response, but when I call this with Ajax I get an error, I didn't find any blog post explain how to use serverless functions with Vuejs, and I'm sure some other people face the same thing.

**Comment by amyegan** [comment]
Hi @sidiDev 👋 Do you have a public repo or minimal reproduction that you can share? That will help me get a better look at what is going wrong
